README.md

```diff
@@ -1,127 +1,50 @@
-
-As such, you do not need to switch APIs when composing your FlowMods and other types. Let's say you wish to build a FlowMod and send it to a switch. Each switch known to the OFSwitchManager has a reference to an OpenFlow factory of the same version negotiated in the initial handshake between the switch and the controller. Simply reference the factory from your switch, create the builder, build the FlowMod, and write it to the switch. The same API is exposed for the construction of all OpenFlow objects, regardless of the OpenFlow version. You will however need to know what you are allowed to do for each OpenFlow version; otherwise, if you for example tell an OpenFlow 1.0 switch to perform some action such as add a Group, which is not supported for it's OpenFlow version, the OpenFlowJ-Loxi library will kindly inform you with an UnsupportedOperationException.
-
-There are some other subtle changes introduced, for the better. For example, many common types such as switch datapath IDs, OpenFlow ports, and IP and MAC addresses are defined by the OpenFlowJ-Loxi library through the DatapathId, OFPort, IPv4Address/IPv6Address, and MacAddress, respectively. You are encouraged to explore org.projectfloodlight.openflow.types, where you will find a wide variety of common types that are now conveniently defined in a single location. Like the objects produced from builders above, all types are immutable.
-
-For more information on how to use the new APIs exposed in Floodlight v1.2, please refer to the OpenFlowJ-Loxi documentation and examples here (https://floodlight.atlassian.net/wiki/display/floodlightcontroller/How+to+use+OpenFlowJ-Loxigen).
-
-There are many more minor details, which can be found in the release notes. I have been grateful to have the support of many Floodlight developers, and together we have worked to provide the highest quality release within a reasonable time frame. I would especially like to thank the following individuals and beta testers for their code contributions and debugging efforts:
-
-https://floodlight.atlassian.net/wiki/display/floodlightcontroller/Authors+and+Contributors
+Floodlight is the leading open source OpenFlow controller. It is [supported by a community of developers](https://floodlight.atlassian.net/wiki/display/floodlightcontroller/Authors+and+Contributors), including a number of engineers from [Big Switch Networks](http://www.bigswitch.com/).
 
-Based on further community feedback, there will be minor releases to address any issues found or enhancements anyone would like to contribute. The mailing list has seen quite an uptick in activity over the last few months! 
+What is OpenFlow?
```

Comment:
Would this be better suited as a wiki page or a FAQ? Just a thought

Reply:
I considered it. Thought it was concise enough to leave here.

Reply:
Sounds good